userspace/libsinsp/parsers.cpp

```diff
@@ -2255,8 +2255,18 @@ void sinsp_parser::parse_execve_exit(sinsp_evt *evt) {
 			auto thread_ptr = thread.lock().get();
 			/* we don't want to remove the main thread since it is the one
 			 * running in this parser!
+			 *
+			 * Also make sure the thread to be removed is not the one
+			 * associated with the event. Under normal conditions this
+			 * should not happen, since the kernel will reassing tid before
```

Comment:
```suggestion
			 * should not happen, since the kernel will reassign tid before
```

Reply:
Thanks! In fact, I think the word with typo perfectly describes what kernel does, when the tid doesn't change :)